scripts/screeps_tencent_batch_rl_runner.py

```
@@ -0,0 +1,783 @@
+        cmd = """
+set -euo pipefail
+cloud-init status --wait >/dev/null 2>&1 || true
+printf 'whoami='; whoami
+printf 'ready='; cat /opt/screeps-batch/READY 2>/dev/null || true; printf '\n'
+printf 'iptables='; sudo iptables -S INPUT | tr '\n' ';'; printf '\n'
+printf 'sshd='; sudo sshd -T 2>/dev/null | egrep '^(passwordauthentication|kbdinteractiveauthentication|pubkeyauthentication|permitrootlogin|allowusers) ' | tr '\n' ';'; printf '\n'
+""".strip()
+        cp = self.ssh_cmd("verify_worker_security", bash_lc(cmd), timeout=180)
+        out = cp.stdout
+        if f"-s {self.args.controller_ip}" not in out and self.args.controller_ip not in out:
+            raise BatchRunError("worker iptables does not show controller-only SSH source")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)